### PR TITLE
Trees

### PR DESCRIPTION
**Pull request by mparkersimms**
[ x] Top-level README “Table of Contents” is updated
[x ] Feature tasks for this challenge are completed
[ x] Unit tests written and passing
[x ] “Happy Path” - Expected outcome
[ x] Expected failure
[ x] Edge Case (if applicable/obvious)
[ x] README for this challenge is complete
[ x] Summary, Description, Approach & Efficiency, Solution
[ x] Link to code
[x ] Picture of whiteboard

### CODE REVIEW

File: utilities/app/src/test/java/utilities/Tree/TreeTester.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+import utilities.tree.Tree;
+import org.junit.*;
+import static org.junit.Assert.*;
+
+import java.sql.Array;
+import java.sql.SQLOutput;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class TreeTester {
```
Comment (Mattpet26) [inline utilities/app/src/test/java/utilities/Tree/TreeTester.java:15]: Fantastic work with your tests!
